frontend/pages/_id/index.vue

```diff
@@ -23,7 +23,10 @@
     <div class="container">
       <div :class="['grid', annotationEnabled ? 'grid--editable' : '']">
         <Results :dataset="dataset" :headerHeight="headerHeight"> </Results>
-        <SideBar :dataset="dataset" />
+        <SideBar :dataset="dataset" :annotationEnabled="annotationEnabled">
+          <TextClassificationMetrics v-if="dataset.task === 'TextClassification'" :dataset="dataset" />
+          <TokenClassificationMetrics v-else :dataset="dataset" :entities="dataset.entities" />
```

Comment:
Could we just pass dataset en refer to `dataset.entities` inside `TokenClassificationMetrics` component?